src/shared/components/palette/collection/read-collection-field.html

```diff
@@ -1,14 +1,27 @@
-<table *ngIf="caseField.value && caseField.value.length" class="collection-field-table">
-  <tbody>
-    <tr *ngFor="let item of caseField.value; let i = index">
-      <td>
-        <ccd-field-read [caseField]="{
+<table *ngIf="caseField.value && caseField.value.length " class="collection-field-table">
+  <tbody *ngIf="isDisplayContextParameterAvailable">
+  <tr>
+    <td>
+      <ccd-field-read [caseField]="{
+              id: caseField.label,
+              label: caseField.label,
+              field_type: caseField.field_type.collection_field_type,
+              display_context_parameter: caseField.display_context_parameter,
```

Comment:
Cannot we just pass the display_context and let ccd-field-read deal with it here rather than splitting it into if else and introducing new display_context_parameter concept?

Reply:
Since we need it in a specific format , that is why I guess, It was done as per the requirements in the related backend stories in definition and data store

Reply:
What I was asking here is more about why are we splitting it into two instead of passing it always to ccd-field-read and handling it there.

Reply:
Ok after a second look I do not think this could be pulled down easily hence it is ok with me.